python/docs/conf.py

```diff
@@ -39,7 +45,6 @@
     "IPython.sphinxext.ipython_directive",
     "nbsphinx",
     "recommonmark",
-    "sphinx.ext.napoleon",
```

Comment:
FYI: this extension was causing havoc with the docs. A lot of things weren't rendering properly, and I would guess it was because of napoleon interacting poorly with Cython-generated docs or a conflict with some other Sphinx extension. The numpydoc extension is sufficient to use on its own nowadays, and napoleon isn't needed here.